src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -795,6 +807,24 @@ public function theCustomerCompletedTheOrder(): void
         $this->objectManager->flush();
     }
 
+    /**
+     * @Given /^there is a (customer "[^"]+") that has placed 2 orders with numbers "([^"]*)" and "([^"]*)"$/
+     */
+    public function thereIsCustomerThatPlacedTwoOrders(
+        CustomerInterface $customer,
+        string $orderNumber1,
+        string $orderNumber2
+    ): void {
+        $order1 = $this->createOrder($customer, $orderNumber1);
+        $order2 = $this->createOrder($customer, $orderNumber2);
+
+        $this->sharedStorage->set('customer', $customer);
+        $this->sharedStorage->set('orders', [$orderNumber1 => $order1, $orderNumber2 => $order2]);
```

Comment:
With the usage of transformers, as explained above, we would not need these shared storage setting here